server/src/main/java/dev/slimevr/vr/trackers/IMUTracker.java

```diff
@@ -456,13 +456,15 @@ protected void adjustToIdentity(Quaternion store) {
 	}
 
 	private void fixGyroscope(Quaternion sensorRotation) {
+		sensorRotation = sensorRotation.clone();
```

Comment:
```suggestion
```

Reply:
The line right below that modifies it, so it needs the clone()
		`sensorRotation.fromAngles(0, sensorRotation.getYaw(), 0);`

Reply:
Could do `sensorRotation = new Quaternion().fromAngles(0, sensorRotation.getYaw(), 0);` if you'd prefer a single line ig

Reply:
Then replace inverse with inverseLocal below

Reply:
:+1: